### DISCUSSION TITLE
Zig 中文社区微信群

### DISCUSSION THREAD

**Discussion by jiacai2050** [zigcc/forum#34] [公告]
![qr code](https://github.com/zigcc/.github/raw/main/weixin.jpg)

每个月初更新一次，如果二维码过期，可以关注『微信公众号』
![qr code](https://github.com/zigcc/.github/raw/main/zig_mp.png)

**Comment by jiacai2050** [comment]
先到先得，过期不候，哈哈！

**Reply by fwx5618177** [reply]
过期了，再发一个吧

**Reply by jiacai2050** [reply]
已经更新

**Comment by FuGangqiang** [comment]
二维码好像过期了，请问是否可以再发一个？

**Reply by jiacai2050** [reply]
我月底再更新一波

**Reply by jiacai2050** [reply]
Updated.

**Comment by dejavudwh** [comment]
@jiacai2050 又过期了，能否再分享一个，感谢！

**Reply by jiacai2050** [reply]
月初更新，可以先关注公众号
